### ISSUE TITLE
Callback function not executing

### ISSUE THREAD

**Issue by sajirdalas** [endlesshack/youtube-video#2]
Been using the library perfectly for a while. Today the callback function is not being called. omgig.com is experimenting similar issue. Did something change youtube-side?

**Comment by mstuart** [comment]
:+1: I'm either getting a Same origin policy error in the browser or it errors out when trying to parse the response.  It makes me think that the endpoint is out-of-date... The URL schema doesn't look the same as their more modern APIs.

**Comment by jwagener** [comment]
yep, looks like youtube changed / dropped the CORS headers for the http://www.youtube.com/get_video_info endpoint. I don't know of any other official or inofficial endpoint that exposes the raw stream url.

kind of makes this library obsolete :(

if you have any idea how to work around it or fix it, let me know.

**Comment by mstuart** [comment]
will do.  thanks for the quick response @jwagener

**Comment by ghost** [comment]
So... this project? it's still working or it's just left there? Any new release soon?
Thank you.

**Comment by jwagener** [comment]
Not working anymore, sorry

Johannes
